crates/runtime/src/lib.rs

```diff
@@ -36,14 +37,53 @@ pub enum Error {
 
     #[snafu(display("Unable to start OpenTelemetry server"))]
     UnableToStartOpenTelemetryServer { source: opentelemetry::Error },
+
```

Comment:
All these errors need to explicitly print the underlying source. I just merged a PR that went through and fixed this for every error in the codebase so you just need to make sure you pull in those changes properly.

Reply:
Thank you - double checked all errors are propogated